requests/status_codes.py

```diff
@@ -87,5 +87,5 @@
 for code, titles in _codes.items():
     for title in titles:
         setattr(codes, title, code)
-        if not title.startswith('\\'):
+        if not title.startswith('\\') and not title.strartswith('/'):
```

Comment:
Additionally, you can write this as `if not title.startswith(('\\', '/'))`, which may be clearer.